gui/default/syncthing/core/syncthingController.js

```diff
@@ -553,10 +569,22 @@ angular.module('syncthing.core')
                 $scope.connectionsTotal = data.total;
 
                 data = data.connections;
+                $scope.recentRemoteAddressCache = {};
                 for (id in data) {
                     if (!data.hasOwnProperty(id)) {
                         continue;
                     }
+                    if (!(id in $scope.idToRemoteAddress)) {
+                        $scope.idToRemoteAddress[id] = "";
+                    }
+                    var port = $scope.findDevice(id).remoteAddressPort.toString();
```

Comment:
This is a nit-pick, so feel free to ignore, but I don't think the toString is needed here.

I think it'd be better if toString happened when the port is used in replaceAddressPort, carryng the expected type as far as we can, because the port is a number everywhere, and in this function it's always starts of as a string, so if the reader misses the toString(), he might do a comparison with a number 0 and introduce a bug.

Reply:
I actually was confused when reviewing because I missed the toString(), hence my comments on the earlier (now resolved comment)